src/app/services/app-state.service.ts

```diff
@@ -82,12 +82,12 @@ export class AppStateService {
 
     window.parent.postMessage(message, "*");
 
-    // Modifies the app"s URL
+    // Modifies the app's URL
     const queryParams = window.location.search?.substr(1).split("&").map((paramString: string) => {
 
       const [key, value] = paramString.split("=");
 
-      if (key.toLowerCase() !== "rowid") {
+      if (key && key.toLowerCase() !== "rowid") {
```

Comment:
Just need to add the empty string comment to prevent future refactors of this condition